src/main/resources/application.properties

```diff
@@ -23,3 +23,8 @@ management.metrics.web.server.request.autotime.enabled=false
 management.metrics.enable.logback=false
 management.metrics.enable.system=false
 management.metrics.enable.spring.integration=false
+
+# caching (override type in application.properties to enable)
+spring.cache.type=none
```

Comment:
disables caching unless overridden in local config

Reply:
Yes, long term I think we'd enable it by default but at least for the initial rollout I wanted it disabled. The alternative is that we enable it by default now and then roll out config changes to disable it on prod envs until we're happy with it.

I'm not tied to either approach - this seemed easier initially.